bdd/features/e2e/E2E-010-cli.feature

```diff
@@ -63,6 +63,19 @@ This feature checks CLI functionalities
         Then I wait for Instance health status to change from 200 to 404
         And host is still running
 
+    @cli
+    Scenario: E2E-010 TC-006 Get log from Instance
+        Given host is running
+        Then I set json format
+        Then I use apiUrl in config
+        When I execute CLI with "seq send ../packages/reference-apps/inert-function.tar.gz" arguments
+        Then I get Sequence id
+        Then I start Sequence
+        And I get Instance id
+        When I execute CLI with "inst log -" arguments without waiting for the end
+        Then confirm instance logs received
```

Comment:
We should be checking for some specific output. What if we got an exception instead of output and mistakenly reported success because we got "something"?

Reply:
This is a CLI test and the above is tested in other tests - you pointed out yourself, that we should make the tests as simple as possible. The `log -` method works.